index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>My Personal Website</title>
+  </head>
+  <body>
+    <header>
+      <h1>Welcome to my personal website</h1>
+      <nav>
+        <ul>
+          <li>Home</li>
+          <a href="#about">
+            <li>About Me</li>
+          </a>
```

Comment:
Would put the a href into the <li> element. See more here: https://developer.mozilla.org/de/docs/Web/HTML/Element/li